build.gradle.kts

```diff
@@ -196,7 +196,9 @@ tasks.register("javaPreCommitPortabilityApi") {
 tasks.register("javaPostCommit") {
   dependsOn(":sdks:java:extensions:google-cloud-platform-core:postCommit")
   dependsOn(":sdks:java:extensions:zetasketch:postCommit")
-  dependsOn(":sdks:java:io:google-cloud-platform:postCommit")
+  dependsOn(":sdks:java:io:debezium:integrationTest")
```

Comment:
These line were removed in #21804 but added back in this PR. Sounds like a rebase issue. Should do a rebase instead of merge.

Reply:
Oh, this is so embarrassing, sorry.
That is correct, it was not intentional.

Reply:
@damccorm will do

Reply:
No problem, I should've caught it too! Happy to quick approve/merge a pr to remove them